openmc/cell.py

```diff
@@ -485,20 +485,23 @@ def get_all_materials(self):
 
         Returns
         -------
-        materials : dict
+        materials : collections.OrderedDict
             Dictionary whose keys are material IDs and values are
             :class:`Material` instances
 
         """
-
         materials = OrderedDict()
         if self.fill_type == 'material':
             materials[self.fill.id] = self.fill
-
-        # Append all Cells in each Cell in the Universe to the dictionary
-        cells = self.get_all_cells()
-        for cell in cells.values():
-            materials.update(cell.get_all_materials())
+        elif self.fill_type == 'distribmat':
```

Comment:
I think that seeing this made me wonder if `'distribmat'` should be considered a fill type in our file format specifications.